qiskit_experiments/data_processing/data_action.py

```diff
@@ -30,46 +30,74 @@ def __init__(self, validate: bool = True):
         self._validate = validate
 
     @abstractmethod
-    def _process(self, datum: Any) -> Any:
+    def _process(self, datum: Any, error: Optional[Any] = None, **options) -> Tuple[Any, Any]:
         """
         Applies the data processing step to the datum.
 
         Args:
             datum: A single item of data which will be processed.
+            error: An optional error estimation on the datum that can be further propagated.
+            options: Keyword arguments passed through the data processor at run-time.
```

Comment:
How will these runtime options work in a DataProcessor workflow when this object is built up in advance?

Reply:
Sounds like this needs some more discussion. In the interest of merging the SVD I removed `**options` here: e7884bc